scss/_main.scss

```diff
@@ -0,0 +1,227 @@
+@font-face {
+    font-family: 'unica_oneregular';
+    src: url('../font/unicaone-regular-webfont.woff2') format('woff2'), url('../font/unicaone-regular-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'varela_roundregular';
+    src: url('../font/varelaround-regular-webfont.woff2') format('woff2'), url('../font/varelaround-regular-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+html {
+    font-size: 10px;
+    box-sizing: border-box;
+    height: 100%;
+}
+
+*,
+*:before,
+*:after {
+    box-sizing: inherit;
+}
+
+body {
+    font-size: 1.4rem;
+    font-family: 'varela_roundregular', sans-serif;
+    color: #444;
+    padding: 0;
+    margin: 0;
+    min-height: 100%;
+    padding: 4rem 2rem;
+    @media (min-width: 480px) {
+        padding: 4rem 4rem;
+    }
+}
+
+main {
+    display: block;
+}
+
+
+h1 {
+    color: #fff;
+    text-align: center;
+    font: 400 4rem/1 'Chewy', cursive;
+    margin-bottom: 0;
+    text-shadow: $text-shadow;
+    @media (min-width: 360px) {
+        font-size: 4.5rem;
+    }
+    @media (min-width: 480px) {
+        font-size: 6rem;
+    }
+}
+
+h2 {
+    margin: 0 0 1rem;
+}
+
+p {
+    margin-top: 0;
+    &:last-child {
+        margin-bottom: 0;
+    }
+}
+
+.container {
+    max-width: 600px;
+    margin: 0 auto;
+}
+
+.box {
+    padding: 2rem;
+    background-color: #ECF0F1;

```

Comment:
Try and use `$variables` for colors where possible in your variables.scss, makes things much more maintanable down the line.